scripts/magic/drc.tcl

```diff
@@ -22,7 +22,7 @@ if { [info exist ::env(MAGIC_DRC_USE_GDS)] && $::env(MAGIC_DRC_USE_GDS) } {
 			lef read $lef_file
 		}
 	}
-	def read $::env(CURRENT_DEF)
+	def read $::env(CURRENT_DEF) -noblockage
```

Comment:
why do we need `-noblockage`? did something break?

Reply:
https://github.com/RTimothyEdwards/open_pdks/issues/258

Reply:
sorry missed it in the PR summary.